rest/blip_sync.go

```diff
@@ -22,10 +22,8 @@ import (
 
 // HTTP handler for incoming BLIP sync WebSocket request (/db/_blipsync)
 func (h *handler) handleBLIPSync() error {
-	// Exit early when the connection can't be switched to websocket protocol.
 	if _, ok := h.response.(http.Hijacker); !ok {
-		base.DebugfCtx(h.ctx(), base.KeyHTTP, "Non-upgradable request received for BLIP+WebSocket protocol")
-		return base.HTTPErrorf(http.StatusUpgradeRequired, "Can't upgrade this request to websocket connection")
+		base.InfofCtx(h.ctx(), base.KeyHTTP, "Non-upgradable request received for BLIP+WebSocket protocol")
```

Comment:
Dropping the `return` here seems incorrect and wasn't in the original `master` PR ?

https://github.com/couchbase/sync_gateway/pull/6423/files?#diff-2a0339ec110fd1d4844ddd3355d1047e6f823464863223982f751af8f4566610R36-R39

Reply:
Thanks for the catch - resolved.